auto_nag/scripts/regression.py

```diff
@@ -50,6 +50,10 @@ def get_bz_params(self, date):
             'emailtype1': 'notequals',
             'email1': 'intermittent-bug-filer@mozilla.bugs',
             'emailreporter1': 1,
+            # And by the wpt-sync bot.
```

Comment:
I'd prefer to have a black-list of emails in config/tools.json.

Reply:
Done